.gitignore

```diff
@@ -0,0 +1,23 @@
+# Specific files/dirs to ignore for this pipeline
+
+#Targets directry
+_targets/
+
+#All output
+*/out/*
```

Comment:
Are you planning to save the crosswalk table to an output folder in the repo?

Reply:
I wasn't originally planning on committing the xwalk table but have opted to in this revision.